packages/components/src/modifiers/hds-code-editor.ts

```diff
@@ -468,6 +468,7 @@ export default class HdsCodeEditorModifier extends Modifier<HdsCodeEditorSignatu
       }
 
       this.editor = editor;
+      (element as HTMLElement & { editor: EditorViewType }).editor = editor;
```

Comment:
alternatively, we could extend the HTMLElement type to include the `editor` property and use this type for the `element` property as well.

```ts
interface HTMLElementWithEditor extends HTMLElement {
  editor?: EditorViewType;
}
```

```suggestion
      (element as HTMLElementWithEditor).editor = editor;
```

Reply:
Great idea. Done.